Classes/Form/Field/Text.php

```diff
@@ -36,6 +36,16 @@ class Text extends Input implements FieldInterface {
 	protected $enableRichText = FALSE;
 
 	/**
+	 * @var string
+	 */
+	protected $renderType;
```

Comment:
I really have no idea about the default $renderType. I've been taking a look in sysext and looks like there are more options now for renderType. If I understood well, looks like ¿maybe wizards going to be deprecated and renderType will sustite at least some of them? Don't know...